<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordManager</class>
 <widget class="QWidget" name="PasswordManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="laDragDrop">
   <property name="geometry">
    <rect>
     <x>525</x>
     <y>200</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ドラッグ＆ドロップ</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>10</y>
     <width>341</width>
     <height>113</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="laClassName">
      <property name="text">
       <string>クラス名</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="liedClassName">
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbPassword">
      <property name="text">
       <string>パスワード</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="liedPassword">
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnDelItem">
   <property name="geometry">
    <rect>
     <x>525</x>
     <y>280</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>削除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFileSave">
   <property name="geometry">
    <rect>
     <x>525</x>
     <y>340</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableItems">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>180</y>
     <width>491</width>
     <height>251</height>
    </rect>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>10</y>
     <width>261</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="laClassList">
      <property name="text">
       <string>クラス一覧</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listClassNames"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnAddClass">
      <property name="text">
       <string>クラス追加</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnUpdateClassInfo">
      <property name="text">
       <string>適用</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnDelClass">
      <property name="text">
       <string>クラス削除</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frmDrag">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>220</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
